Add binding calc

danieleongari opened the pull request:
This contains work chains to compute binding energies with FF and DFT.
Therefore, it is related to PR to:

* aiida-lsmo: https://github.com/lsmo-epfl/aiida-lsmo/pull/23
* aiida-cp2k: https://github.com/aiidateam/aiida-cp2k/pull/66

danieleongari commented:
Tested and rebased, please merge!

yakutovicha commented:
tests are failing, have a look [here](https://travis-ci.org/yakutovicha/aiida-raspa/builds/637494243?utm_source=github_status&utm_medium=notification). Can you fix that?

danieleongari commented:
* removed some of the misleading outputs (e.g. Temperature and Pressure that were 0.0, because possible they are only related to some advanced calculation)
* updated tests adding one for widom calculation (that returns many different outputs)
* updated examples to run faster and to output some more meaningful property like the number of particles

yakutovicha commented on aiida_raspa/utils/base_parser.py:
```diff
@@ -21,21 +21,19 @@ def float(number):  # pylint: disable=redefined-builtin
+    #("Average Heat Capacity", "framework_heat_capacity", (1, 2, 4), 0), # misleading property!
+    ("Enthalpy of adsorption:", "enthalpy_of_adsorption", (1, 4, 3), 4),
+    ("Tail-correction energy:", "tail_correction_energy", (1, 2, 4), 0)
+    #("Total energy:$", "total_energy", (1, 2, 4), 0), # not important property!
```
Can you please explain why those properties are misleading/not important? Otherwise, it is fine with me.

danieleongari replied:
* Heat capacity is meaningless for rigid particles/frameworks
* Temperature and pressure are giving 0.0: not clear why
* The total energy of a FF system is useless: anyway, I already parsed the host/ads and ads/ads contributes of the energy that can be useful in case of comparing interactions.
